src/hooks/useStablecoinPrice.ts

```diff
@@ -15,13 +16,16 @@ export const STABLECOIN_AMOUNT_OUT: { [chainId: number]: CurrencyAmount<Token> }
   [SupportedChainId.ARBITRUM_ONE]: CurrencyAmount.fromRawAmount(USDC_ARBITRUM, 10_000e6),
   [SupportedChainId.OPTIMISM]: CurrencyAmount.fromRawAmount(DAI_OPTIMISM, 10_000e18),
   [SupportedChainId.POLYGON]: CurrencyAmount.fromRawAmount(USDC_POLYGON, 10_000e6),
+  [SupportedChainId.CELO]: CurrencyAmount.fromRawAmount(CUSD_CELO, 10_000e18),
+  //TODO remove after resolving token price (redux-multicall) issue?
```

Comment:
What's the issue here?

Reply:
By removing line 21, we won't be able to get the USD price of tokens in testnet. I don't think it's a big deal, will remove.